_data/glossary/wg.yaml

```diff
@@ -0,0 +1,9 @@
+id: wg
+name: WG (Working Group)
+tags:
+- community
+short-description: |
+  Facilitates the discussion and/or implementation of a short-lived, narrow, or decoupled project for a committee, SIG, or cross-SIG effort.
+long-description: |
+  They are a way of organizing people to accomplish a discrete task and are relatively easy to create and deprecate when inactive.
```

Comment:
Working groups are a way of organizing people to accomplish a discrete task, and are relatively easy to create and deprecate when inactive.